apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -275,6 +276,13 @@ protected static CommandLine parseArgs(String[] args) {
             .longOpt("tableFilter")
             .desc("Regexp for filtering tables, defaults to .*")
             .build());
+    options.addOption(
+        Option.builder(null)
+            .required(false)
+            .hasArg()
+            .longOpt("timeFilter")
+            .desc("Time in hour for filtering older tables, defaults to 1")
```

Comment:
Nice PR @rohitkum2506 I would recommend a much higher default. Maybe 3 days?

Reply:
Considering SE and OFD TTL configs, default 3 days makes sense. 
For retention job, we should override the `timeFilter` to hourly granularity. `1 hour` is supposedly the lowest, but we can discuss if a higher value (24hr) is suitable considering hourly tables in our system.